package.json

```diff
@@ -47,6 +47,7 @@
     ]
   },
   "dependencies": {
+    "json-logic-js": "^2.0.2",
```

Comment:
I'm lost. Wasn't this PR all about new test cases? Where are they now that you closed the PR?